File: Day_4/Condition.java
import java.io.*;
import java.math.*;
import java.security.*;
import java.text.*;
import java.util.*;
import java.util.concurrent.*;
import java.util.regex.*;

public class Solution {

    private static final Scanner scanner = new Scanner(System.in);

    public static void main(String[] args) {
        int n = scanner.nextInt();
        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
        String output="";
        if (n%2 == 1) {
            output = "Weird";
        }
        
        else {
            if (n > 20) {
                output = "Not Weird";
            }
            else {
                if (n >= 6) {
                    output = "Weird";
                }
                else {
                    output = "Not Weird";
                }
            }
        }
        
        System.out.print(output);
        
        scanner.close();
    }
}

File: Day_4/Condition.py
#!/bin/python3

import math
import os
import random
import re
import sys



if __name__ == '__main__':
    N = int(input())
    output = ''
    if (N%2 == 1):
        output = 'Weird';
    else:
        if (N > 20):
            output = 'Not Weird';
        else:
            if(N >= 6):
                output = 'Weird';
            else:
                output = 'Not Weird';
    print(output)